docs/logs/logs-alerting.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[[xpack-logs-alerting]]
+== Logs alerting
```

Comment:
This seems to lead to an empty page for the section intro:

![image](https://user-images.githubusercontent.com/973741/80952069-5cb59a80-8df9-11ea-8a16-4d2c091a8096.png)

How about keeping this on one page? I think that could be achieved using `[float]` markers (see suggestions below). :thinking: